packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -887,6 +888,34 @@ const useSuggestedConfig = (config: ModelConfiguration) => {
 	knobs.value.timeseries = amr.metadata?.timeseries ?? {};
 	knobs.value.initialsMetadata = amr.metadata?.initials ?? {};
 	knobs.value.parametersMetadata = amr.metadata?.parameters ?? {};
+
+	// Update output port:
+	const listOfConfigIds: string[] = props.node.outputs.map((output) => output.value?.[0]);
+	// Check if this output already exists
+	if (config.id && listOfConfigIds.includes(config.id)) {
+		// Select the existing output
+		const output = props.node.outputs.find((ele) => ele.value?.[0] === config.id);
+		emit('select-output', output?.id);
+	}
+	// If the output does not already exist
+	if (config.id && !listOfConfigIds.includes(config.id)) {
```

Comment:
I'd start this function with
```
if (!config.id) {
 logger.error("Model configuration not found")
 return
}

const state = cloneDeep(props.node.state);
...
```

Then this can be simplified to:
```
if (listOfConfigIds.includes(config.id)) {
		// Select the existing output
		const output = props.node.outputs.find((ele) => ele.value?.[0] === config.id);
		emit('select-output', output?.id);
}
// If the output does not already exist
else { ...
```

Reply:
Fair point, thank you!